packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -2,38 +2,25 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// TODO(darrenaustin): Needed for all the unused token values. Need to find
+// a better solution than turning off the lint for the whole file.
+// ignore_for_file: unused_field
```

Comment:
This is no longer needed with the new way you're doing things right?

Reply:
D'oh. Thanks, I missed that. If only we had a lint for unused lints 😄.